package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
I understand `frontend/package-lock.json`, I don't know what is the purpose of this file though. Did you do anything special or it's just created by npm install? I have a gut feeling this signals that we have to fix npm config somehow, I just don't know how. I might be wrong. Let me know if this is intentional. :)

Reply:
That was just created via npm install, nothing intentional!

Reply:
If it's created by npm, we might as well leave it in, I reckon :)